packages/plugins/src/api/index.js

```diff
@@ -77,6 +143,21 @@ export function registerPlugin( name, settings ) {
  *
  * @param {string} name Plugin name.
  *
+ * @example <caption>ES5</caption>
+ * ```js
+ * // Using ES5 syntax
+ * var unregisterPlugin = wp.plugins.unregisterPlugin;
+ *
+ * unregisterPlugin( 'plugin-name' );
+ * ```
+ *
+ * @example <caption>ESNext</caption>
+ * ```js
+ * // Using ESNext syntax
+ * const { unregisterPlugin } = wp.plugins;
+ * unregisterPlugin( 'plugin-name' );
```

Comment:
Nit: Missing an empty line to align with the other example.

Reply:
Fixed at 9d814ab and the automated process reminded me that I should also update the README. So I did at d15f919 Starting to :heart: it. Also: :sweat_smile: